emspy/query/fltquery.py

```diff
@@ -123,13 +124,16 @@ def __split_expr(self, expr):
 		import re
 
 		for pattern in ['[=!<>]=?'] + list(sp_ops.keys()):
-			a = re.search(pattern, expr)
-			if a is not None:
+			a = re.search(
+				"(.*)\s+(%s)\s+(.*)\s+(%s)\s+(.*)|(.*)\s+(%s)\s+(.*)".replace('%s', pattern),
+				expr
+			)
+			if a is not None and len([s for s in a.groups() if s is not None]):
```

Comment:
len([s for s in a.groups() if s is not None]) gets repeated twice here . Should it get its own variable name?  What is the significance?  Count of ... ?